frontend/src/components/Profile/Profile.tsx

```diff
@@ -49,9 +48,12 @@ function Profile() {
       />
       <Container onClick={() => setIsopen(false)}>
         <ProfilePage
+          // eslint-disable-next-line no-underscore-dangle
+          id={data._id}
           email={data.email}
           displayName={data.displayName}
           createdAt={data.createdAt}
+          imageUrl
         />
         {/* <Map onOpenBigPopup={onOpenBigPopup} /> */}
```

Comment:
i think we can remove this part

Reply:
i create frontend part write feedback on my pr